pkg/io/sftpclient/sftpclient.go

```diff
@@ -57,7 +62,7 @@ func NewClient(source string) Client {
 	if p := sshCreds["port"]; p != "" {
 		port, err = strconv.Atoi(p)
 		if err != nil || port < 1 || port > 65535 {
-			logrus.Fatalf("Port number (%s) is wrong.", p)
+			return Client{}, errors.New("Port number " + p + " is wrong.")
```

Comment:
Normally I'd do `return Client{}, fmt.Errorf("Port number %s is wrong.", p)`

Reply:
cool, I didn't know fmt.Errorf return error